kbs/docs/id-key.md

```diff
@@ -0,0 +1,84 @@
+# ID-key Plugin
```

Comment:
what is the rationale behind the name? it's not obvious to me what an ID-Key is, maybe spell that out in the doc.

Reply:
It was sort-of a reference to @fitzthum's [The Mystery of the KBS Identity](https://gist.github.com/fitzthum/f3124f846e6ecbc066c53c6d31668979). Since the NEK was encrypted from a known key provisioned on a known-trusted source, the very fact that it could be decrypted indicated that the client was communicating with the "true" KBS server.

Not saying a **good** reference, just a reference :smile: . If you feel another name is more appropriate, I'm open to changing it.

Reply:
but isn't the identity of the KBS settled by the KBS cert that we inject into a guest in init-data?